### PR TITLE
Add radar support

### PR DESCRIPTION
**Pull request by mrafidashti**
In this change I'm adding a data class for the radar data. This class is mostly similar to LidarData except that instead of a single radar frame it includes radar data for the entire sequence. It reads from and writes to a radar .npy file, and right now I mostly need it to upload the radar data for sequences and drives to dropbox. Each file includes the following information:

- radar range
- azimuth angle
- elevation angle
- range rate (detection velocity relative to the radar sensor)
- amplitude (signal to noise ratio / SNR)
- validity (detection validity 0 or 1)
- mode (radar mode, can be 0, 1, and 2, depends on ego speed and changes the FOV.)
- quality (detection quality, can be 0:low, 1: medium, and 2: high.)
- scan index (which radar frame the detection comes from)
- timestamp (in nanoseconds)

### CODE REVIEW

File: zod/data_classes/info.py [hunk 1]
```diff
@@ -130,3 +131,8 @@ def get_camera_frames(
 
     def get_lidar_frames(self, lidar: Lidar = Lidar.VELODYNE) -> List[LidarFrame]:
         return self.lidar_frames[lidar]
+
+    def get_radar_frames(self, radar: Radar) -> RadarFrames:
+        if self.radar_frames is None:
+            raise ValueError("No radar frames available.")
```
Comment (atonderski) [inline zod/data_classes/info.py:137]: maybe also inform the user that they should download the new dataset (maybe even give them the command here for maximal user friendlyness)? Or can this also happen when the data is downloaded due to some missing files?

**Review by atonderski — COMMENTED** [review]
I will do some testing to make sure backwards compatibility

File: zod/data_classes/info.py (hunk 1 above)
Comment (mrafidashti) [reply zod/data_classes/info.py:137]: This shouldn't happen when the data is downloaded. I'll update the message here.

Comment (atonderski) [reply zod/data_classes/info.py]: nice

**Review by atonderski — APPROVED** [review]
lgtm

**Pull request merged**